var/spack/repos/builtin/packages/cmake/package.py

```diff
@@ -24,7 +24,7 @@ class Cmake(Package):
 
     tags = ['build-tools']
 
-    executables = ['^cmake$']
+    executables = ['^cmake$', '^cmake3$']
```

Comment:
```suggestion
    executables = ['^cmake[0-9]*$']
```
Wouldn't this be more robust (e.g., cmake 2.8, 4.0, etc.)?